cmd/lncli/main.go

```diff
@@ -2,28 +2,33 @@ package main
 
 import (
 	"fmt"
+	"io/ioutil"
 	"os"
 	"path/filepath"
 	"strings"
+	"time"
+
+	"gopkg.in/macaroon-bakery.v1/bakery/checkers"
+	"gopkg.in/macaroon.v1"
```

Comment:
We shouldn't be using `gopkg.in` here when importing the packages, and also when specifying the new dependancies in the `glide.yaml` file. `gopkg.in` was initially created in a time before vendoring support was added to Go. It's a way to create a repo that pins a project against a particular version. But since we use `glide` for dependency management, it isn't needed IMO. 

Instead we can pin directly against the two repositories directly: 
  * https://github.com/go-macaroon/macaroon 
  * https://github.com/go-macaroon-bakery/macaroon-bakery